sites/next.skeleton.dev/src/content/docs/components/tab/meta.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Tab
```

Comment:
The page section should be named "Tabs" (plural) instead of "Tab" singular.

This is one of those odd situations in English where the single group entity is typically referred to in plural. This is because it's a group of things. Feel free to compare to a few other libraries to see what I mean.

Make sure to update the meta info, directory (for the route, etc